src/lib.rs

```diff
@@ -120,8 +121,11 @@ impl<K, V, S> LruCache<K, V, S> where K: Eq + Hash, S: HashState {
     /// assert_eq!(cache.get(&1), None);
     /// assert_eq!(cache.get(&2), Some(&"c"));
     /// ```
-    pub fn get(&mut self, k: &K) -> Option<&V> {
-        self.map.get_refresh(k)
+    pub fn get<Q: ?Sized>(&mut self, k: &Q) -> Option<&V>
```

Comment:
Actually, I'm not sure what we should do here. It's weird for `get` (rather than `get_mut`) to return `&mut V`, but since calling this requires `&mut self`...